packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js

```diff
@@ -13,12 +13,14 @@ jest.mock('@strapi/helper-plugin', () => ({
   useLibrary: () => ({ components: { 'media-library': jest.fn() } }),
 }));
 
-const setup = (props) =>
+const setup = ({ value, ...props }) =>
   render(
     <Blocks
       intlLabel={{ id: 'blocks', defaultMessage: 'blocks type' }}
       name="blocks-editor"
-      value={blocksData}
+      value={value}
+      hint="blocks description"
+      placeholder={{ id: 'blocksPlaceholder', defaultMessage: 'blocks placeholder' }}
       onChange={jest.fn()}
       {...props}
```

Comment:
I don't think you need to treat value differently, spreading props handles it

Reply:
yeahh 👍